frontend/packages/dev-console/src/components/pipelines/modals/common/MultipleKeySelector.tsx

```diff
@@ -39,23 +38,24 @@ const MultipleKeySelector: React.FC<MultipleKeySelectorProps> = ({
                 const fieldKey = `${name}.${index}.${item.key}`;
                 return (
                   <div className="form-group odc-multiple-key-selector__item" key={fieldKey}>
-                    <DropdownField
-                      name={`${name}.${index}.key`}
-                      title="Select a key"
-                      items={keys}
-                      fullWidth
-                    />
-                    <InputField
-                      name={`${name}.${index}.path`}
-                      type={TextInputTypes.text}
-                      placeholder="Enter a path"
-                      isDisabled={!item.key}
-                    />
-                    <div
-                      className={cx('odc-multiple-key-selector__deleteButton', {
-                        '--disabled': items.length <= 1,
-                      })}
-                    >
+                    <Grid hasGutter>
```

Comment:
Maybe go with a Flex? [Responsive design](https://www.patternfly.org/v4/documentation/react/layouts/flex#responsive-layout-modifiers) for mobile would help a bit more.

![image](https://user-images.githubusercontent.com/8126518/91610970-3d162500-e948-11ea-965f-be19b26637fc.png)

Reply:
![image](https://user-images.githubusercontent.com/20724543/91738273-d6a23880-ebcd-11ea-9db4-ac7f784c932e.png)
Done